### ISSUE TITLE
Looping video load shows blank state

### ISSUE THREAD

**Issue by gravnetic** [pojome/elementor#4659]
Here is a solution featured on YouTube - https://stackoverflow.com/questions/48652139/remove-black-loading-screen-from-youtube-embed-looped-video?rq=1

I am guessing this is not an issue in Vimeo, but when you load a video background the video shows a black screen on buffering which is not ideal and again a single frame of black on the video loop which is certainly not ideal.

Thank you in advance.

Heres is a page with a looping video - https://wavelengthspr.com/ although I will probably upload it to Vimeo to see if it corrects the issue.

**Comment by bainternet** [comment]
Thanks for reporting,
Please follow guidelines for opening a new issue/feature request, so we can help you better.
This report is missing key information: 

* Prerequisites.
* Steps to replicate.
* Isolating the problem.
* System info.

**Comment by bainternet** [comment]
Closing for lack of activity,

Feel free to reopen if the issue persists.

**Comment by gravnetic** [comment]
The factors that you have outlined don't apply exactly...or in other words I have provided all the details necessary to replicate the issue.

Prerequisites: Embedded YouTube video link of a length that is short enough to see it buffer and reload.
Steps to replicate:  Add a video to the Elementor background, a default feature.
Isolate the problem:  See stackoverflow article for the problem and solution.
System info does not apply at it occurs on all devices utilizing browser/internet systems.

I find these requisites annoying when they don't apply and moreover feel that the closed ticket indicates a lack of attention to the issue which was more or less clearly outlined.
